### ISSUE TITLE
Error after Docker container upgrade.

### ISSUE THREAD

**Issue by cgraefe** [wallabag/wallabag#3622]
### Issue details

I have been using the official Wallabag Docker container version 2.2.3 for a while. After the upgrade to version 2.3.2, Wallabag ceased to function and displays only white page in the browser. The container log shows this error message:

```
2018/04/03 08:08:37 [error] 232#232: *3 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught RuntimeException: Setting "piwik_enabled" couldn't be found. in /var/www/wallabag/vendor/craue/config-bundle/Util/Config.php:182
Stack trace:
#0 /var/www/wallabag/vendor/craue/config-bundle/Util/Config.php(77): Craue\ConfigBundle\Util\Config->createNotFoundException('piwik_enabled')
#1 /var/www/wallabag/vendor/craue/config-bundle/Twig/Extension/ConfigTemplateExtension.php(96): Craue\ConfigBundle\Util\Config->get('piwik_enabled')
#2 /var/www/wallabag/var/cache/prod/twig/b2/b2ed076ea096208791515657de1c778185907df95538a5a4b6efa66c2e6112ab.php(85): Craue\ConfigBundle\Twig\Extension\ConfigTemplateExtension->getSetting('piwik_enabled')
#3 /var/www/wallabag/vendor/twig/twig/lib/Twig/Template.php(432): __TwigTemplate_34806d58992d1e92415396d8c0dc93a7e2a6b69ebaadd6037c5428dc75cea4a2->doDisplay(Array, Array)
#4 /var/www/wallabag/vendor/twig/twig/lib/Twig/Template.php(403): Twig_Template->displayWithErrorHandling(Array, Array)
#5 /var/www/wallabag/var/cache/prod/twig/07/07cfc2e" while reading response header from upstream, client: 172.17.0.4, server: _, request: "GET /login HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "XXXXXXXXXXXX"
172.17.0.4 - - [03/Apr/2018:08:08:37 +0000] "GET /login HTTP/1.1" 500 31 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:52.0) Gecko/20100101 Firefox/52.0"
```

### Environment

* wallabag version (or git revision) that exhibits the issue:
2.3.2

* How did you install wallabag? Via `git clone` or by downloading the package?
docker run --name wallabag -e "MYSQL_ROOT_PASSWORD=my-secret-pw" -e "SYMFONY__ENV__DATABASE_DRIVER=pdo_mysql" -e "SYMFONY__ENV__DATABASE_HOST=mariadb" -e "SYMFONY__ENV__DATABASE_PORT=3306" -e "SYMFONY__ENV__DATABASE_NAME=wallabag" -e "SYMFONY__ENV__DATABASE_USER=wallabag" -e "SYMFONY__ENV__DATABASE_PASSWORD=wallapass" -p 80:80 wallabag/wallabag:2.3.2

* Last wallabag version that did not exhibit the issue (if applicable):
2.2.3

* php version:
7.1.2

* OS:
alpine:edge

* type of hosting (shared or dedicated):
Docker

* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL):
MariaDB

### Steps to reproduce/test case

1. Start a MariaDB Docker container.
2. Start and configure Wallabag container version 2.2.3 with MariaDB container as storage.
3. Add at least one link to Wallabag. (Maybe not needed.)
4. Stop Wallabag container.
5. Create and start a new version 2.3.2 container.
6. Migrade DB using:
`docker exec -t wallabag /var/www/wallabag/bin/console doctrine:migrations:migrate --env=prod --no-interaction`
7. Navigate browser to new docker container.

**Comment by cgraefe** [comment]
Ok, right after I submitted this issue, I was able to resolve the problem for myself using these steps:

1. Create and start container again, this time adding the SYMFONY__ENV__DOMAIN_NAME variable. I suppose this wasn't required in version 2.2?

2. Clear Symfony cache:
`docker exec -t wallabag /var/www/wallabag/bin/console cache:clear --env=prod`

3. Fix cache dir ownership:
`docker exec -t wallabag chown -R nobody:nobody /var/www/wallabag/var/cache`
